common/djangoapps/enrollment/views.py

```diff
@@ -411,18 +411,29 @@ def post(self, request):
             # have to check if it is inactive or not, because if it is, we are still upgrading if the mode is different,
             # and either path will re-activate the enrollment.
             #
-            # Only server-to-server calls will currently be allowed to modify the mode for existing enrollments. All
-            # other requests will go through add_enrollment(), which will allow creating of new enrollments, and
-            # re-activating enrollments
+            # Only server-to-server calls are currently allowed to deactivate or modify the mode of existing
```

Comment:
This comment should be above the `has_api_key_permissions` line. As is, it is somewhat confusing here in the middle of the view.